test/integration/default/controls/_mapdata_spec.rb

```diff
@@ -0,0 +1,21 @@
+# frozen_string_literal: true
+
+# Replace per minion strings
+replacement = {
+  hostname: system.hostname
+}
+
+# Keep only first 2 digits from Ubuntu finger
+mapdata_file = "_mapdata/#{system.platform[:finger].split('.').first}.yaml"
+
+# Load the mapdata from profile https://docs.chef.io/inspec/profiles/#profile-files
+mapdata_dump = inspec.profile.file(mapdata_file) % replacement
+
+control '`map.jinja` YAML dump' do
+  title 'should contain the lines'
+
+  describe file('/tmp/salt_mapdata_dump.yaml') do
+    it { should exist }
```

Comment:
These two lines depend on the conclusion of https://github.com/saltstack-formulas/template-formula/pull/203#issuecomment-684942520 -- i.e. could be `describe yaml` instead.

Reply:
Yes, I used `eq` for now since it's what is actually used for [openssh-formula](https://github.com/saltstack-formulas/openssh-formula).

It will be changed according to the result of the discussion.